sns_ik_lib/CMakeLists.txt

```diff
@@ -6,9 +6,9 @@ include(CheckCXXCompilerFlag)
 CHECK_CXX_COMPILER_FLAG("-std=c++11" COMPILER_SUPPORTS_CXX11)
 CHECK_CXX_COMPILER_FLAG("-std=c++0x" COMPILER_SUPPORTS_CXX0X)
 if(COMPILER_SUPPORTS_CXX11)
-    set(CMAKE_CXX_FLAGS "-std=c++11 -O2")
+    set(CMAKE_CXX_FLAGS "-std=c++11 -O2 -Wall -Werror")
```

Comment:
Hmmm... Good point. How can I look up the best practices for this? I did a quick google search and made little progress. 
Perhaps I should change it to `-Wall -Wextra` but let builds pass with warnings?

My goal here was to prevent pull requests that added questionable code to the project, assuming that everyone would build their project before making a pull-request.

I'll leave it for now, but I'll defer to you on your next pass through the review.

Reply:
I removed the error on warning flag, but kept `-Wall`.